languages.toml

```diff
@@ -1781,3 +1781,14 @@ language-server = { command = "bass", args = ["--lsp"] }
 [[grammar]]
 name = "bass"
 source = { git = "https://github.com/vito/tree-sitter-bass", rev = "501133e260d768ed4e1fd7374912ed5c86d6fd90" }
+
+[[language]]
+name = "wat"
+scope = "source.webassembly"
+comment-token = ";;"
+file-types = ["wast", "wat"]
```

Comment:
It looks like wast is a separate grammar within the tree-sitter-wasm repo. It looks like a small superset of the wat parser though so I don't have strong feelings about separating these out into separate languages. What do you think: should `wast` be separate?

Reply:
I bundled them together thinking that `.wast` was historically used for the true text format that is defined in the spec, and that usage might still linger around. Splitting the two is no big deal, if you're fine with having the less commonly used `wast` language bundled by default :+1: